lxd/container_lxc.go

```diff
@@ -4945,7 +5057,27 @@ func (c *containerLXC) Update(args db.ContainerArgs, userRequested bool) error {
 	}
 
 	// Diff the devices
-	removeDevices, addDevices, updateDevices, updateDiff := oldExpandedDevices.Update(c.expandedDevices)
+	removeDevices, addDevices, updateDevices, updateDiff := oldExpandedDevices.Update(c.expandedDevices, func(oldDevice types.Device, newDevice types.Device) []string {
```

Comment:
@stgraber this is the approach I've gone with to 'wire up' the new device interface into the device live update system such that each device can provide the config fields it feels are candidates for being live updated.